observability/cluster_observability_operator/cluster-observability-operator-release-notes.adoc

```diff
@@ -13,6 +13,43 @@ The {coo-short} complements the built-in monitoring capabilities of {product-tit
 
 These release notes track the development of the {coo-full} in {product-title}.
 
+
+[id="cluster-observability-operator-release-notes-1-1_{context}"]
+== {coo-full} 1.1
+
+// Back fill advisory when it is created by KOnflux
+// The following advisory is available for {coo-full} 1.1:
+//
+// * link:https://access.redhat.com/errata/RHSA-2025:????[RHEA-2025:??? {coo-full} 1.1]
+
+
+[id="cluster-observability-operator-1-1-new-features-enhancements_{context}"]
+=== New features and enhancements
```

Comment:
Should the dev preview of Perses [Operator] be mentioned? https://issues.redhat.com/browse/COO-265

Reply:
we aren't allowed talk about dev previews in OCP docs :-(